src/app/clients/create-address/create-address.component.ts

```diff
@@ -14,33 +15,57 @@ import { ClientsService } from 'app/clients/clients.service';
 
 })
 export class CreateAddressComponent implements OnInit {
+  /** Client id */
   id: number = undefined;
+  /** Snapshot user id */
   paramsSubscription: Subscription;
+  /** Client address */
   clientAddress: any = undefined;
-  value: any = undefined;
-  mymodel: any = undefined;
-  addressTypeId: any = undefined;
-  addressStateId: any = undefined;
-  addressCountryId: any = undefined;
-
-  address: any = undefined;
-
-  constructor(private route: ActivatedRoute, private clientService: ClientsService) {}
-
-  @ViewChild('f') addressForm: NgForm;
-
+  /** add new address form. */
+  newAddressForm: FormGroup;
+  /**
+   *
+   * @param route active route to snapshot user
+   * @param clientService Client service
+   * @param formBuilder Form builder
+   * @param router for navigation
+   */
+  constructor(private route: ActivatedRoute,
+              private router: Router,
+              private clientService: ClientsService,
+              private formBuilder: FormBuilder) {}
 
+  /**
+   * Creates the new address form
+   */
   ngOnInit() {
+    this.createNewAddressForm();
```

Comment:
```suggestion
    this.createAddressForm();
```
Update everywhere accordingly.